style.css

```diff
@@ -382,3 +382,123 @@ header img {
   font-style: normal;
   font-weight: bold;
 }
+
+.contact_section {
+  background-image: url("images/contact\ form\ shapes-mobile@2x.png");
+  background-size: cover;
+  border-top-left-radius: 80px;
+  background-color: #6070ff;
+  padding-top: 50px;
+  width: 375px;
+  height: 779px;
+}
+
```

Comment:
- [ ] As the previous reviewer said, you must stick to the design as much as possible using the Figma Template provided.
- [ ] Please, try to remove the margin from the left and the bottom of your contact form to improve the UI design.
- [ ] I would recommend not using a fixed width to make your form responsive. It would be better to use "min-width" instead of "width" or try to use "width: 100%".
- Your design:
<img width="194" alt="image" src="https://user-images.githubusercontent.com/77190203/150353731-bd27fc67-eba2-4fe5-8083-e9d7f66afbb7.png">

Reply:
Hello @medaminedev66. Please guide us on how to proceed because we have implemented the margin and the  width based on the Figma design and values were also got from the same design (Template1).  

Please note that; 
- We implemented all the changes requested by the first reviewer.
- We are only left with one more code review request

Reply:
Hi @Berucan,
Please are you available now maybe we can get on a call, maybe I can be of help to resolve the issue before you re-submit?